dependency-license-format.json

```
@@ -0,0 +1,10 @@
+{
+	"licenseFile": false,
+	"licenseText": false,
+	"licenseModified": false,
```

```suggestion
    "licenseFile": false,
    "licenseText": false,
    "licenseModified": false,
```
Fix to the indentation.